.github/native-tests.json

```diff
@@ -92,8 +92,8 @@
         },
         {
             "category": "HTTP",
-            "timeout": 110,
-            "test-modules": "elytron-resteasy, resteasy-jackson, elytron-resteasy-reactive, resteasy-mutiny, resteasy-reactive-kotlin/standard, vertx, vertx-http, vertx-web, vertx-web-jackson, vertx-graphql, virtual-http, rest-client, rest-client-reactive, rest-client-reactive-stork, rest-client-reactive-multipart, websockets, management-interface, management-interface-auth, mutiny-native-jctools",
+            "timeout": 113,
```

Comment:
I think you are a bit too optimistic about CI speed :). We usually add 5 minutes per added module.